src/app/angularjs/version-stamp.component.ts

```diff
@@ -31,20 +30,20 @@ export const versionStampComponent = {
 
     <md-divider id="version-stamp-divider"></md-divider>
     <div class="version-container">
-      <span class="version"><img src="/assets/angularjs.svg" alt="AngularJS icon" height="24px"> AngularJS v{{ $ctrl.versions.angularjs.full }} ({{ $ctrl.versions.angularjs.codeName }})</span>
-      <span class="version"><img src="/assets/angularjs-material.png" alt="AngularJS Material icon" height="24px"> AngularJS Material v{{ $ctrl.versions.md }}</span>
-      <span class="version"><img src="/assets/angular.svg" alt="Angular icon" height="24px"> Angular v{{ $ctrl.versions.angular }}</span>
-      <span class="version"><img src="/assets/angular-cdk.svg" alt="Angular CDK icon" height="24px"> Angular CDK v{{ $ctrl.versions.cdk }}</span>
-      <span class="version"><img src="/assets/angular-material.svg" alt="Angular Material icon" height="24px"> Angular Material v{{ $ctrl.versions.mat }}</span>
+      <span class="version"><img src="/assets/angularjs.svg" alt="" height="24px"> AngularJS v{{ $ctrl.versions.angularjs.full }} ({{ $ctrl.versions.angularjs.codeName }})</span>
+      <span class="version"><img src="/assets/angularjs-material.png" alt="" height="24px"> AngularJS Material v{{ $ctrl.versions.md }}</span>
+      <span class="version"><img src="/assets/angular.svg" alt="" height="24px"> Angular v{{ $ctrl.versions.angular }}</span>
+      <span class="version"><img src="/assets/angular-cdk.svg" alt="" height="24px"> Angular CDK v{{ $ctrl.versions.cdk }}</span>
+      <span class="version"><img src="/assets/angular-material.svg" alt="" height="24px"> Angular Material v{{ $ctrl.versions.mat }}</span>
```

Comment:
Let's also specify the height in CSS.
(BTW, the height property should be set to an [integer value, without a unit][1] :grin:

[1]: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/img#:~:text=The%20intrinsic%20height%20of%20the%20image%2C%20in%20pixels.%20Must%20be%20an%20integer%20without%20a%20unit.